examples/rust/jsonschema/src/lib.rs

```diff
@@ -0,0 +1,90 @@
+wit_bindgen_rust::export!("extension.wit");
+use boon::{Compiler, Schemas};
+use serde_json::Value;
+
+struct Extension;
+
+const DEFAULT_SCHEMA_URL: &str = "http://tmp/schema.json";
+
+impl extension::Extension for Extension {
+    fn match_schema(schema_str: String, instance_str: String) -> bool {
+        let schema: Value = serde_json::from_str(schema_str.as_str()).unwrap();
+        let instance: Value = serde_json::from_str(instance_str.as_str()).unwrap();
+
+        let mut schemas = Schemas::new();
+        let mut compiler = Compiler::new();
+        compiler.add_resource(DEFAULT_SCHEMA_URL, schema).unwrap();
+        let sch_index = compiler.compile(DEFAULT_SCHEMA_URL, &mut schemas).unwrap();
+        let result = schemas.validate(&instance, sch_index);
+        if result.is_err() {
```

Comment:
nit:  this block could be abbreviated to just:
`!result.is_err()`

Same with the one below.